File: apps/s-2-admin/src/api/destinationCalendar/DestinationCalendar.ts
import { User } from "../user/User";
import { Booking } from "../booking/Booking";
import { EventType } from "../eventType/EventType";
import { Credential } from "../credential/Credential";

export type DestinationCalendar = {
  id: number;
  integration: string;
  externalId: string;
  user?: User | null;
  booking?: Booking | null;
  eventType?: EventType | null;
  credential?: Credential | null;
};

File: apps/cactus-demo/src/APIs/DestinationCalendar/Dtos/DestinationCalendar.cs
namespace CactusDemo.APIs.Dtos;

public class DestinationCalendar
{
    public int Id { get; set; }

    public string Integration { get; set; }

    public string ExternalId { get; set; }

    public int? Booking { get; set; }

    public int? Credential { get; set; }

    public int? EventType { get; set; }

    public int? Users { get; set; }
}
